### ISSUE TITLE
Fonts, pricing, text display

### ISSUE THREAD

**Issue by Andrewdwatters1** [Andrewdwatters1/invictus-auto-detailing-web#8]
All prices should be preceded by a currency sign. Text should be consistent throughout and use psychologically "soft" text for pricing and "buy" actions. Font colors should be checked for contrast and accessibility to ensure maximum readability and accessibility.
